pkg/controllers/webhook/utils_test.go

```diff
@@ -390,16 +384,25 @@ func TestBuildRulesWithOperations(t *testing.T) {
 				},
 			},
 			mapResourceToOpnType: map[string][]admissionregistrationv1.OperationType{
-				"Namespace": {},
-				"Pod":       {webhookCreate, webhookUpdate},
+				"Namespace":  {},
```

Comment:
What's the corresponding operations for namespace?

Reply:
That was an extra resource I should have removed.